### ISSUE TITLE
Endpoint Security Considerations

### ISSUE THREAD

**Issue by Scott-Borecki** [tvaroglu/spot_me_backend#36]
One other thing to consider is security.  We don't want a malicious user to be able to delete records when they shouldn't be deleted.  But maybe that's another discussion :)

_Originally posted by @Scott-Borecki in https://github.com/tvaroglu/spot_me_backend/pull/34#discussion_r711623211_

**Comment by tvaroglu** [comment]
I agree... personally it makes me nervous that we're exposing documentation this thoroughly for anything other than GET requests... but I understand it is part of the project deliverables.
